src/Tribe/CSV_Importer/RSVP_Importer.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+
+/**
+ * Class Tribe__Tickets__CSV_Importer__RSVP_Importer
+ */
+class Tribe__Tickets__CSV_Importer__RSVP_Importer extends Tribe__Events__Importer__File_Importer {
+
+	/**
+	 * @var array
+	 */
+	protected $required_fields = array( 'event_name', 'ticket_name' );
+
+	/**
+	 * @var array
+	 */
+	protected static $event_name_cache = array();
+
+	/**
+	 * @var array
+	 */
+	protected static $ticket_name_cache = array();
+
+	/**
+	 * @var Tribe__Tickets__RSVP
+	 */
+	protected $rsvp_tickets;
+
+	/**
+	 * @var bool|string
+	 */
+	protected $row_message = false;
+
+	/**
+	 * The class constructor proxy method.
+	 *
+	 * @param Tribe__Events__Importer__File_Importer|null $instance The default instance that would be used for the type.
+	 * @param Tribe__Events__Importer__File_Reader        $file_reader
+	 *
+	 * @return Tribe__Tickets__CSV_Importer__RSVP_Importer
+	 */
+	public static function instance( $instance, Tribe__Events__Importer__File_Reader $file_reader ) {
```

Comment:
Doesn't look like we use/need the `$instance` argument right now - can we remove it, or do you think we'll need it in the future?